RealEstate/apps/core/templates/core/categories.html

```diff
@@ -0,0 +1,266 @@
+$(".slide").slider({
+	formatter: function(value) {
+          return jsWeights[value] || "";
+	}
+});
+
+// Provides user interaction when saving through ajax.
+$(".slide").change($.debounce( 700, slideChange ));
+function slideChange (slideEvt) {
+	var timer;
+	clearTimeout(timer);
+	var categoryID = $(this).attr("id").toString().replace(/category_/i, "");
+	$("#save-status").text("saving...").css("color", "#888");
+	$.ajax({
+		type: "POST",
+		data: {
+			csrfmiddlewaretoken: token,
+			category: categoryID,
+			weight: slideEvt.value.newValue,
+			type: "update"
+		},
+		success: function(data) {
+			timer = setTimeout(function(){
+			    $("#update_"+categoryID).stop().show();
+        		$("#update_"+categoryID).fadeOut(2000);
+        		$("#save-status").text("All changes saved.").show().css("color", "#333");
+			}, 1500);
+		}
+	});
+};
+
+// Deletes category given an ID
+function deleteCategory(deleteID) {
+    $.ajax({
+		type: "POST",
+		data: {
+			csrfmiddlewaretoken: token,
+			category: deleteID,
+			type: "delete"
+		},
+		success: function(data) {
+		    $("#li_"+data.id).remove();
+		}
+	});
+}
+
+// Gets category summary and description, given id.
+function editCategory() {
+    $.ajax({
+        type: "GET",
+        data: {
+            csrfmiddlewaretoken: token,
+            category: catID
+        },
+        success: function(data) {
+            $('input[name="edit_summary"]').val(data.summary);
+            $('input[name="edit_description"]').val(data.description);
+        }
+    });
+}
+
+// Shows edit and delete icons on div mouseenter.
+$(".category_row").mouseenter( function() {
+    catID = $(this).attr("id").toString().replace(/row_/i, "");
+    $("#delete_"+catID).fadeIn("slow", function(){
+        $("#delete_"+catID).click( function(){
+            var sum = $(this).parent().text();
+            $("#confirmDelete").data("id", catID );
+            $("#confirmDelete .btn-danger").text("Delete " + sum);
+        });
+    });
+    $("#edit_"+catID).fadeIn("slow", function(){
+        $("#edit_"+catID).click( $.debounce(300, editCategory))
+        $('input[name="catID"]').val(catID);
+    });
+}).mouseleave( function() {
+    var id = $(this).attr("id").toString().replace(/row_/i, "");
+    $("#edit_"+id).stop().fadeOut("fast");
+    $("#delete_"+id).stop().fadeOut("fast");
+});
+
+// Handles sizing of columns and breaks columns at appropriate times.
+$(window).on("resize", function () {
+	var divWidth = $(".row").width();
+	if(divWidth < 570){
+		$(".rating").css("width", "90%");
+		$(".help-text").hide();
+		$(".rating-col").css("min-width", "0");
+	}
+	else if(divWidth < 750){
+		$(".help-text").show();
+		$(".rating").css("width", "90%");
+		$(".rating-col").css("min-width", "500px");
+	}
+	else{
+		$(".help-text").show();
+		$(".rating").css("width", "60%").css("margin", "0 auto");
+		$(".rating-col").css("min-width", "500px");
+	}
+}).resize();
```

Comment:
This resize function feels very similar to the one on the eval page.  Refactorable?